.konflux/applications/serverless-operator-release-next/components/imagerepositories/openshift-knative-eventing-release-next-knative-eventing-apiserver-receive-adapter.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ImageRepository
+metadata:
+  annotations:
+    image-controller.appstudio.redhat.com/update-component-image: "true"
+  labels:
+    appstudio.redhat.com/application: serverless-operator-release-next
+    appstudio.redhat.com/component: knative-eventing-apiserver-receive-adapter-release-next
+  name: serverless-operator-release-next-knative-eventing-apiserver-receive-adapter
```

Comment:
this is 75 chars long

Reply:
we could follow the component name scheme https://github.com/openshift-knative/eventing/blob/c418629db6c37def043f88cee2f7ef4ccaee1db3/.konflux/applications/serverless-operator-release-next/components/openshift-knative-eventing-release-next-knative-eventing-apiserver-receive-adapter.yaml#L9 with only the version